udata/core/access_type/api.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint
+
+from udata.api import API, api
+from udata.core.access_type.utils import get_inspire_limitation_labels
+
+blueprint = Blueprint("access_type", __name__, url_prefix="/api/1/access_type")
+ns = api.namespace("access_type", "Access type related operations")
+
+
+@ns.route("/reason_categories/", endpoint="reason_categories")
+class ReasonCategoriesAPI(API):
+    @api.doc("reason_categories")
+    def get(self):
+        """List all INSPIRE limitation reason categories"""
```

Comment:
I wouldn't mention INSPIRE here, seems it isn't known by all and may be extended with other categories
```suggestion
        """List all limitation reason categories"""
```